ql/src/test/results/clientpositive/llap/limit_join_transpose.q.out

```diff
@@ -1207,7 +1223,6 @@ limit 1 offset 1
 POSTHOOK: type: QUERY
 POSTHOOK: Input: default@src
 #### A masked pattern was here ####
-0	val_0	0	val_0
```

Comment:
Result set disappeared.

Reply:
Good catch, it might be because there is an offset=1 belongs to the limit. The Limit is pushed through the LOJ, but the original one is also kept. The first limit emits 1 row, but the second emits 0 because of offset=1.